doc/langref.html.in

```diff
@@ -2180,16 +2180,23 @@ or
       Unlike normal structs, {#syntax#}packed{#endsyntax#} structs have guaranteed in-memory layout:
       </p>
       <ul>
-        <li>Fields remain in the order declared, least to most significant.</li>
+        <li>Fields are arranged in the order declared, from the least to the most significant bits of a backing integer.</li>
         <li>There is no padding between fields.</li>
-        <li>Zig supports arbitrary width {#link|Integers#} and although normally, integers with fewer
-        than 8 bits will still use 1 byte of memory, in packed structs, they use
-        exactly their bit width.
+        <li>The backing integer has the same bit width as the fields' total bit width.</li>
+        <li>The backing integer is subject to the same rules as any integer, including {#link|alignment|Alignment#}, 
+        having a maximum bit count of 65535, and the host endianness.
+        On a big endian system, the first declared field will have the highest memory address, and on a little endian system, the lowest.
```

Comment:
I don't think this part of the documentation is a good place to document how endianness works.  This isn't really a documentation of what packed structs are.

Reply:
I don't see the harm in having this information here.
It also isn't just documentation "how endianness works", but how it interacts with struct layout here. The language used to be specified/implemented differently in this regard, and this point is often asked about (see inciting issue https://github.com/ziglang/zig/issues/20647 - andrew didn't close it, but reflagged it from bug to documentation issue ).
Ultimately the wording and scope is up to andrew to decide. I don't think trimming it now (on anyone else's opinion) is helpful.

Reply:
The GitHub comment system makes this unclear because of the context window: I was only referring to line 2188 here.   Endianness is not really part of the user interface of packed structs, it's a fact about the backing integers. 

The point of review is to come up with a better patch, so I disagree that it isn't helpful to do it. However, given the lack of consensus on this point, I think it's fine to err on the side of leaving it in.